library/Vanilla/Web/Page.php

```diff
@@ -110,21 +114,32 @@ abstract class Page implements InjectableInterface, CustomExceptionHandler {
      * @param WebpackAssetProvider $assetProvider
      * @param BreadcrumbModel $breadcrumbModel
      * @param ContentSecurityPolicyModel $cspModel
+     * @param ConfigurationInterface $config
      */
     public function setDependencies(
         SiteMeta $siteMeta,
         \Gdn_Request $request,
         \Gdn_Session $session,
         WebpackAssetProvider $assetProvider,
         BreadcrumbModel $breadcrumbModel,
-        ContentSecurityPolicyModel $cspModel
+        ContentSecurityPolicyModel $cspModel,
+        ConfigurationInterface $config
```

Comment:
Could we put that favicon in SiteMeta? I've been trying to isolate `Page` from using the configuration directly.

Reply:
Works for me. I'm in favor of avoiding adding another dependency to this class.

Favicon and the "mobile address bar color" both need to see the config, so they've both been moved into `SiteMeta`.